karamel-core/src/main/java/se/kth/karamel/backend/machines/SshMachine.java

```diff
@@ -183,8 +183,12 @@ private void runTask(Task task) {
           } else {
             try {
               task.collectResults(this);
+              // If this task is an experiment, try and download the experiment results
+              if (cmd.getCmdStr().contains("experiment") && cmd.getCmdStr().contains("json")) {
```

Comment:
This is the name of the recipe. We can't make it too long - people won't like that. We could call it "karamelXXX" - but i think that would not be appreciated either. We could strengthen this to make it an experiment prefix on the name. A prefix like 'test' is great for junit. But 'expr' is too short and unclear, imo. The UI will auto-generate the first recipe with the name 'experiment', so this will work out of the box for most users.